### PR TITLE
UI

### LINKED ISSUE #3: Set up UI
Create storyboards for UI and setup basic UI

### PR DESCRIPTION
**Pull request by FabiolaSaga**
Closes #3

### CODE REVIEW

File: Slango/Slango/ButtonShadow.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+//  Slango
+//
+//  Created by Fabiola S on 2/28/20.
+//  Copyright © 2020 Fabiola Saga. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class ButtonShadow: UIButton {
```
Comment (dimitribouniol) [inline Slango/Slango/ButtonShadow.swift:12]: You might want to name this `ShadowedButton` to be more clear that this class is a button and not a shadow.

File: Slango/Slango/HomeViewController.swift [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        // Do any additional setup after loading the view.
+    }
+    
+    // MARK: - UI
+    
+    func setGradientBackgroundColor() {
+        
+        let gradientLayer = CAGradientLayer()
+        gradientLayer.frame = view.bounds
+        gradientLayer.colors = [#colorLiteral(red: 0, green: 0.3529411765, blue: 0.737254902, alpha: 1).cgColor, #colorLiteral(red: 0, green: 0.7573584914, blue: 1, alpha: 1).cgColor ]
```
Comment (dimitribouniol) [inline Slango/Slango/HomeViewController.swift:29]: Color literals won’t work with DarkMode. You can use asset catalog colors instead though!

File: Slango/Slango/WordDetailViewController.swift [hunk 3]
```diff
@@ -9,14 +9,32 @@
...
         // Do any additional setup after loading the view.
     }
     
+    // MARK: - UI
+    
+    func setGradientBackgroundColor() {
+        
+        let gradientLayer = CAGradientLayer()
+        gradientLayer.frame = view.bounds
+        gradientLayer.colors = [#colorLiteral(red: 0, green: 0.3529411765, blue: 0.737254902, alpha: 1).cgColor, #colorLiteral(red: 0, green: 0.7573584914, blue: 1, alpha: 1).cgColor ]
```
Comment (dimitribouniol) [inline Slango/Slango/WordDetailViewController.swift:29]: Same here

**Review by dimitribouniol — APPROVED** [review]
Looks good! See my notes on using the asset catalog for your colors too.

File: Slango/Slango/HomeViewController.swift (hunk 2 above)
Comment (FabiolaSaga) [reply Slango/Slango/HomeViewController.swift:29]: Thanks for the suggestion! Will try that right now.

File: Slango/Slango/ButtonShadow.swift (hunk 1 above)
Comment (FabiolaSaga) [reply Slango/Slango/ButtonShadow.swift:12]: Done!

**Pull request merged**